src/main/java/uk/gov/companieshouse/company/profile/service/CompanyProfileService.java

```diff
@@ -173,29 +168,24 @@ public void updateInsolvencyLink(String contextId, String companyNumber,
             } else {
                 Updated updated = new Updated(LocalDateTime
                         .now().truncatedTo(ChronoUnit.SECONDS),
-                        contextId, "company_delta");
+                        DataMapHolder.getRequestId(), "company_delta");
                 cpDocument.setUpdated(updated);
             }
 
-            ApiResponse<Void> response = companyProfileApiService.invokeChsKafkaApi(
-                    contextId, companyNumber);
+            ApiResponse<Void> response = companyProfileApiService.invokeChsKafkaApi(companyNumber);
 
             HttpStatus statusCode = HttpStatus.valueOf(response.getStatusCode());
 
             if (statusCode.is2xxSuccessful()) {
-                LOGGER.info(String.format("Chs-kafka-api CHANGED invoked successfully for company number %s", companyNumber),
-                        DataMapHolder.getLogMap());
                 if (cpDocument.getVersion() == null) { // Update a legacy document
                     mongoTemplate.save(new UnversionedCompanyProfileDocument(cpDocument));
                 } else { // Update a versioned document
                     companyProfileRepository.save(cpDocument);
                 }
-                LOGGER.info(String.format("Company profile is updated in MongoDB with company number %s", companyNumber),
-                        DataMapHolder.getLogMap());
+                LOGGER.info("Company profile is updated in MongoDB", DataMapHolder.getLogMap());
             } else {
-                LOGGER.error(String.format("Chs-kafka-api CHANGED NOT invoked successfully for company number %s. Response code %s.",
-                                companyNumber, statusCode.value()), new Exception("Chs-kafka-api CHANGED NOT invoked"),
-                        DataMapHolder.getLogMap());
+                LOGGER.error(String.format("Chs-kafka-api CHANGED NOT invoked successfully. Response code %s.",
```

Comment:
This message is confusing at a glance. Should use clear wording, e.g. "failed" instead of "not invoked successfully"